site/docs/v1/tech/shared/_client-side-api-keys.adoc

```diff
@@ -0,0 +1,17 @@
+When interacting with FusionAuth from a client side application, such as a React or Angular front end, you have a few options.
+
+If you are authenticating the user, use the standard link:/docs/v1/tech/oauth/[OAuth Authorization Code grant].
+You can either forward the browser or webview to FusionAuth or use an iframe.
+
+If you are using one of the APIs which does not require an API key, such as the link:/docs/v1/tech/apis/users#start-forgot-password-workflow[change password API], you can interact directly with the API.
+These APIs tend to be more limited in functionality when no API key is presented, but may suit your needs.
+
+If you need to use an API requiring authentication from client code, such as the link:/docs/v1/tech/apis/registrations[registration API], you have two options:
+
+* Create an API key with extremely limited permissions and distribute it.
+Make sure you threat model the ramifications of someone stealing the key and using it.
```

Comment:
Can we rephrase this to make it more approachable to the un-initiated? 

Something like:

> Stealing the key when used on the client side is trivial. Make sure you have carefully considered the risk and result of someone stealing and using the key outside of your application.